Library/Formula/gnome-icon-theme.rb

```diff
@@ -18,12 +15,28 @@ class GnomeIconTheme < Formula
   depends_on "gtk+3" => :build # for gtk3-update-icon-cache
   depends_on "icon-naming-utils" => :build
   depends_on "intltool" => :build
+  depends_on "librsvg" => :build
 
   def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
-                          "--enable-icon-mapping",
                           "GTK_UPDATE_ICON_CACHE=#{Formula["gtk+3"].opt_bin}/gtk3-update-icon-cache"
     system "make", "install"
   end
+
+  test do
+    # This checks that a -symbolic png file generated from svg exists
+    # and that a file created late in the install process exists.
+    # Someone who understands GTK+3 could probably write better tests that
+    # check if GTK+3 can find the icons.
+    (testpath/"test.sh").write <<-EOS.undent
+    #!/usr/bin/env bash
+    if [ -r "#{share}/icons/Adwaita/96x96/status/weather-storm-symbolic.symbolic.png" ] && [ -r "#{share}/icons/Adwaita/index.theme" ]; then
+        exit 0;
+    else
+        exit 1;
+    fi
+    EOS
+    system "bash", "./test.sh"
+  end
```

Comment:
You can just write e.g. `assert (share/"icons/Adwaita/96x96/status/weather-storm-symbolic.symbolic.png").exist?` instead of shelling out to a bash script.